### ISSUE TITLE
Can we have a release?

### ISSUE THREAD

**Issue by mohsen0** [trussworks/terraform-aws-config#92]
https://github.com/trussworks/terraform-aws-config/compare/v4.0.2...master

```Warning: Interpolation-only expressions are deprecated

  on .terraform/modules/account_region.aws_config/config-rules.tf line 27, in data "template_file" "aws_config_ami_approved_tag":
  27:   template = "${file("${path.module}/config-policies/ami-approved-tag.tpl")}"

Terraform 0.11 and earlier required all non-constant expressions to be
provided via interpolation syntax, but this pattern is now deprecated. To
silence this warning, remove the "${ sequence from the start and the }"
sequence from the end of this expression, leaving just the inner expression.

Template interpolation syntax is still used to construct strings from
expressions when the template includes multiple interpolation sequences or a
mixture of literal strings and interpolations. This deprecation applies only
to templates that consist entirely of a single interpolation sequence.
```

**Comment by eeeady** [comment]
I... thought I did that... I'm sorry. I'll look at that.

**Comment by eeeady** [comment]
Sorry about that. Just published v4.0.3
